src/components/scenes/SendScene.js

```diff
@@ -448,7 +446,8 @@ class SendComponent extends React.PureComponent<Props, State> {
   }
 
   renderSelectFioAddress() {
-    const { hiddenTilesMap = {}, guiMakeSpendInfo, isSendUsingFioAddress } = this.props
+    const { isSendUsingFioAddress } = this.props
```

Comment:
`const { isSendUsingFioAddress, route } = this.props`, and then you can use `route` on the next line without the `this.props.` bit.